cron/poll_read_issue_comments.py

```diff
@@ -224,12 +226,16 @@ def handle_comment(api, issue_comment):
             command = db_fields["command"]
             body = "> {command}\n\nVote failed".format(command=command)
             gh.comments.edit_comment(api, settings.URN, resp_id, body)
+    else:
+        __log.debug("comment: {comment} is not a command, " +
+                    "parsed: {parsed}".format(comment=comment_text, parsed=orig_parsed))
 
-
+        
```

Comment:
Yeah i did earlier, but this fix is in several PRs and I don't want to have a merge conflict